web/src/components/common/Header.tsx

```diff
@@ -32,19 +36,35 @@ export const Header = (): JSX.Element => {
     axios
       .get('version.txt')
       .then((response: AxiosResponse<string>) => {
-        const responseSubString = response.data.substring(
-          response.data.indexOf('tag:')
-        )
-        setVersion(
-          responseSubString.substring(
-            responseSubString.indexOf('v') + 1,
-            responseSubString.indexOf(',')
-          )
+        const versionFile: { [key: string]: string } = Object.fromEntries(
+          response.data.split('\n').map((line) => {
+            return line.split(': ')
+          })
         )
+        setVersionFile(versionFile)
       })
       .catch(() => null)
   }, [])
 
+  const VersionText = () => {
+    if (versionFile.hash === '' || versionFile.date === '') {
+      return <p>Version: Null</p>
+    } else {
+      const link = (
+        <a
```

Comment:
https://eds-storybook-react.azurewebsites.net/?path=/docs/typography-typography--link might be better

Reply:
Oh nice! Was looking for exactly this and did not know eds had it. Will implement.